### PR TITLE
Revamp landing page with improved design

### PR DESCRIPTION
**Pull request by jackveiga**
This is a PR with the third design iteration for the landing page.

- Improve the overall design of the page
- Illustrations to bring more life to the page
- Show how to get started with Recall

Also, I've started to exchange tachyons with tailwind. All views and components in this PR are now with tailwind.

### CODE REVIEW

**Comment by jackveiga** [comment]
I still need to fix an issue with eslint and vue-typer before being able to proceed

File: src/components/SmallCard.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    SmallCardItems: {
```
Comment (alvesjtiago) [inline src/components/SmallCard.vue]: Prop names should start with lower case and follow camelCase 😊

Comment (jackveiga) [reply src/components/SmallCard.vue]: Sorry my bad, missed this one 😅

**Comment by jackveiga** [comment]
This PR is now ready for review

**Pull request merged**